flatpak-warning.txt

```diff
@@ -0,0 +1,46 @@
+
+https://www.flathub.org
+
+------------------------------------------------------------------------------------
+| Warning: You are running an unofficial Flatpak version of Visual Studio Code !!! |
+|          Visual Studio Code Insiders is also an experimental beta release, and   |
+|          not fully tested and stable! Please switch back to the release channel  |
+|          if you are experiencing issues with this program.                       |
+------------------------------------------------------------------------------------
+
+Please open issues under: https://github.com/flathub/com.visualstudio.code/issues
```

Comment:
It should be pointing to https://github.com/flathub/com.visualstudio.code.insiders

Reply:
Oh, good catch. That was left over from when it may have been a branch in the main repo. Fixed, thanks!